src/test/java/org/opentripplanner/graph_builder/module/GtfsGraphBuilderModuleTest.java

```diff
@@ -20,8 +20,6 @@
 
 public class GtfsGraphBuilderModuleTest {
 
-    private static final HashMap<Class<?>, Object> _extra = new HashMap<>();
-
     private GtfsModule builder;
```

Comment:
Variable `builder` should be declared inside each test (it is assigned in each test each time).